smoke-test/smoke-test.sh

```diff
@@ -28,13 +28,14 @@ grep 'function createApplication' "$ROLLUP_OUT_DIR"/bundle.mjs
 # tsconfig using moduleResolution: nodenext. Let's run it before the others
 # since this is the "pickiest" of the tests.
 pushd nodenext
-  tsc --build .
+  npx tsc --build .
```

Comment:
Is this a necessary change? Is this protecting against if you haven't "npm install"ed in the directory? I think the smoke test will be broken anyway if so. Or maybe this is related to `direnv` config (`.envrc`) using `layout node` ?

Reply:
On my environment this was broken because I don't have `tsc` installed globally.
`npx tsc` is the proper way to execute a node_modules bin from shell.
As far as I can tell it seemed to work in the smoke test in the CI.
<img width="584" alt="image" src="https://github.com/apollographql/apollo-server/assets/4157103/bfa88a3c-3c8a-4251-83da-757156cdc88d">

I don't have a strong opinion and can revert, but I thought it'd be a good change to enable the community to run the smoke-tests

Reply:
No harm with this addition just wanted to understand it. Agree with keeping your change here 👍